3.processing_features/scripts/2.pycytominer_singlecell_pipelines.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Perform single-cell pycytominer pipelines
+# 
+# Following single-cell curation with cytotable, we create single-cell profiles by applying the following steps:
+# 
+# 1. annotation
+# 2. normalization
+# 3. feature_selection
+# 
+# Additionally, we create bulk profiles following feature selection.
+# We call this "Cameron's Method".
+# 
+# 4. Aggregate (to form bulk, after single-cell processing)
+
+# In[1]:
+
+
+import pathlib
+import yaml
+import pprint
+
+import pandas as pd
+
+from pycytominer import aggregate, annotate, normalize, feature_select
+from pycytominer.cyto_utils import load_profiles, output, infer_cp_features
+
+
+# In[2]:
+
+
+# Set constants
+feature_select_ops = [
+    "variance_threshold",
+    "correlation_threshold",
+    "blocklist",
+]
+
+# Columns to remove prior to single-cell aggregation via cameron's method
+cameron_unwanted_aggregate_cols = {"Object", "Parent", "Site", "Image"}
```

Comment:
After looking into this, it looks like Cameron removes columns with these string, not starting with these strings, that makes more sense. For example, this removes Metadata_ImageNumber, Metadata_Site, Metadata_Cells_Number_Object_Number, etc.

But I pose the question, why these columns specifically? Is it because these columns would cause issues when trying to perform the aggregation?

Reply:
> Is it because these columns would cause issues when trying to perform the aggregation?

That's exactly right! Their aggregation calculation is either meaningless or impossible to calculate